admin_manual/configuration_server/reverse_proxy_configuration.rst

```diff
@@ -135,6 +135,9 @@ or
   rewrite ^/\.well-known/carddav https://$server_name/remote.php/dav/ redirect;
   rewrite ^/\.well-known/caldav https://$server_name/remote.php/dav/ redirect;
 
+When using NGINX Proxy Manager, the entry proxy_hide_header Upgrade; must be added in the Advanced Settings
+of the proxy host under "Custom Nginx Configuration", otherwise mobile devices (Ipad, iphone etc.) will simply receive the Error Message "Connection Closed".
```

Comment:
```suggestion
of the proxy host under *"Custom Nginx Configuration"*, otherwise mobile devices (iPad, iPhone etc.) will simply receive the Error Message "Connection Closed".
```